crates/forge/tests/e2e/running.rs

```diff
@@ -240,8 +240,21 @@ fn with_print() {
         Running 0 test(s) from src/lib.cairo
         Running 1 test(s) from tests/test_print.cairo
         original value: [123], converted to a string: [{]
-        original value: [6381921], converted to a string: [aaa]
         original value: [3618502788666131213697322783095070105623107215331596699973092056135872020480]
+        original value: [6381921], converted to a string: [aaa]
+        original value: [12], converted to a string: []
+        original value: [1234]
+        original value: [123456]
+        original value: [1233456789]
+        original value: [123345678910]
+        original value: [0], converted to a string: []
+        original value: [10633823966279327296825105735305134080]
+        original value: [2], converted to a string: []
+        original value: [11], converted to a string: []
+        original value: [1234]
+        original value: [123456]
+        original value: [123456789]
+        original value: [12345612342]
```

Comment:
Why do some of them have and some don't have the `converted to a string` part?

Reply:
Depends if they are convertable to ascii - for example 1234 is 0x4D2, first byte is D2 = 210 > 127 (out of range of ascii)